### PR TITLE
Fixed #25996 -- Revised performance section in URLs topic doc. 

### PR DESCRIPTION
**Pull request by ralphreid**
This PR closes Ticket: https://code.djangoproject.com/ticket/25996
Attempts to resolves PR comments in https://github.com/django/django/pull/6879/files

### CODE REVIEW

**Comment by ralphreid** [comment]
Hiya @carltongibson , 

This is a first attempt.

Ticket comments suggested adding insights from the work related to this PR and the dispatcher performance. 
https://github.com/django/django/pull/5578

From your perspective, is this reasonable now. I am aware of the age on this ticket and did not want go off piste too far.
Thanks 😄

**Review by carltongibson — COMMENTED** [review]
Hey @ralphreid — this looks good. 

Let's leave it to see if others comment, but I quite like it: _minimal_ works, especially given https://github.com/django/django/pull/6879/files#r70154501. 

Can you please squash, and adjust the commit message to ≈match the adjusted PR title. ([Have a read of the committing guidelines](https://docs.djangoproject.com/en/dev/internals/contributing/committing-code/#committing-guidelines)) 

Then grab another one! 👍 

🏅

File: docs/topics/http/urls.txt [hunk 1]
```diff
@@ -317,8 +317,9 @@ default argument for ``num``, ``1``. If the second pattern matches,
 Performance
 ===========
 
-Each regular expression in a ``urlpatterns`` is compiled the first time it's
-accessed. This makes the system blazingly fast.
+Django processes regular expressions in the ``urlpatterns`` list
+which is compiled the first time it's accessed. Subsequent requests use the
+cached configuration via the URL resolver.
```
Comment (felixxm) [inline docs/topics/http/urls.txt:322]: `blazingly` can now be removed from `docs/spelling_wordlist`

**Review by David-Wobrock — APPROVED** [review]
I like it, short and to the point 👍

**Comment by felixxm** [comment]
@ralphreid Thanks :+1: Welcome aboard :boat:

**Review by felixxm — APPROVED** [review]

**Comment by ralphreid** [comment]
> @ralphreid Thanks 👍 Welcome aboard ⛵

Thanks greatly for the welcome!

**Pull request merged**